plugins/catalog-backend/src/ingestion/processors/BitbucketDiscoveryProcessor.ts

```diff
@@ -140,12 +140,15 @@ export class BitbucketDiscoveryProcessor implements CatalogProcessor {
     options: ProcessOptions,
   ): Promise<ResultSummary> {
     const { client, location, integration, emit } = options;
-    const { catalogPath: requestedCatalogPath } = parseUrl(location.target);
+    const { catalogPath: requestedCatalogPath } = parseUrl(
+      location.target,
+      integration.config.baseUrl,
```

Comment:
@benjdlambert, is this implementation of the integration object what you envisioned based on your comment on https://github.com/backstage/backstage/pull/8815#issuecomment-1011844430?

Reply:
Yep!